setup.py

```diff
@@ -26,7 +26,6 @@ def run_tests(self):
     url="http://www.pacificclimate.org/",
     author="James Hiebert",
     author_email="hiebert@uvic.ca",
-    # TODO: Should we adhere to policy: no version numbers in setup.py requirements?
```

Comment:
Don't you still need to upgrade the `modelmeta` requirement here. This package will break using `modelmeta<0.3.0`, right?

Reply:
:facepalm: